app/logger/logger_config_gcp.yaml

```
@@ -20,4 +20,4 @@ root:
   level: WARNING
   handlers: [console_handler]
 
-disable_existing_loggers: False
+disable_existing_loggers: False
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)